efsity/src/main/java/org/smartregister/command/PublishFhirResourcesCommand.java

```diff
@@ -41,6 +43,13 @@ public class PublishFhirResourcesCommand implements Runnable{
             description = "path to env.properties file")
     String propertiesFile;
 
+    @CommandLine.Option(
+      names = {"-vr", "--validate-resource"},
+      description =
+        "-vr or --validate-resource - (Optional) whether to validate FHIR resources before publishing or not. Optional boolean - default is `true`",
```

Comment:
nitpick: no need to repeat the names in the description, just so it matches with all the rest
also the tab space is a bit off